crates/data_components/src/kafka.rs

```diff
@@ -265,6 +265,11 @@ impl KafkaConsumer {
         config
             .set("group.id", group_id.clone())
             .set("bootstrap.servers", &kafka_config.brokers)
+            .set("retry.backoff.ms", "1000")
```

Comment:
In addition to this, should we allow tuning these via parameters on the dataset config?